clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -300,18 +307,18 @@ public void testJoinGroupInvalidGroupId() {
     public void testNormalJoinGroupLeader() {
         final String consumerId = "leader";
 
-        subscriptions.subscribe(singleton(topicName), rebalanceListener);
+        subscriptions.subscribe(Pattern.compile("test.*"), rebalanceListener);
```

Comment:
Not sure why this was changed. Do we have "normal" subscription covered elsewhere?